packages/kbn-search-connectors/types/connector_stats.ts

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { ConnectorStatus } from '..';
+
+export interface ConnectorStats {
```

Comment:
This type is duplicated in a few places. Is it possible to use only one of them as I don't see a difference between them.

Reply:
Hi @efegurkan , I only have `ConnectorStats` defined in this file. Did yo mean other type/class?